bin/Makefile.am

```diff
@@ -53,13 +54,19 @@ _fvwm-perllib.1: fvwm-perllib
 	mv $$TMP/$$NAME.1 .; \
 	rm -rf $$TMP)
 
+FvwmCommand: FvwmCommand.in
+	rm -f $@
+	$(SED) -e 's/__FVWM3_VERSIONINFO__/@VERSION@@VERSIONINFO@/' $(top_srcdir)/bin/FvwmCommand.in > $@-t
+	mv $@-t $@
+
```

Comment:
This entire block can go, because the substitution of `@VERSION-@VERSIONINFO` can be handled via autotools in `configure.ac`.  So what you need to do in that file is something like this -- essentially, add `bin/FvwmCommand` to the list of files in the `AC_OUTPUT` block:

```
diff --git a/configure.ac b/configure.ac
index c2805b57c..4047369f0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1492,6 +1492,7 @@ dnl       bin/fvwm-menu-xlock.1
 dnl    bin/fvwm-perllib.1
 dnl
        bin/Makefile
+       bin/FvwmCommand
        bin/FvwmPrompt/Makefile
        bin/fvwm-config
        bin/fvwm-perllib
```